optics-core/src/Optics/AffineTraversal.hs

```diff
@@ -21,6 +25,13 @@ type AffineTraversal s t a b = Optic An_AffineTraversal NoIx s t a b
 -- | Type synonym for a type-preserving affine traversal.
 type AffineTraversal' s a = Optic' An_AffineTraversal NoIx s a
 
+-- | Type synonym for a type-modifying van Laarhoven affine traversal.
+type AffineTraversalVL s t a b =
+  forall f. Functor f => (forall r. r -> f r) -> (a -> f b) -> s -> f t
```

Comment:
this is not the van Laarhoven representation. Correct one is `forall f. PointedFunctor f => ...`. This is naming issue, so some other name should be used for this type alias

Reply:
True. I figured it's close enough though. It's a similar problem to `foldVL` function.

I'm not sure what the name should be though. Any suggestions?

Reply:
I created #114 for tracking this. It can stay like that for the moment, I'd like to have access to affine traversal specific stuff from this PR in #113 to figure out `zoom`, so I'm going to merge this tomorrow.